views/about.erb

```diff
@@ -0,0 +1,40 @@
+<html>
```

Comment:
With each nested HTML tag, you want to add a level of indentation. So starting with the `body` tag, then carrying that throughout all of your tags.